chimney/src/main/scala-2/io/scalaland/chimney/internal/compiletime/ResultsPlatform.scala

```diff
@@ -0,0 +1,7 @@
+package io.scalaland.chimney.internal.compiletime
+
+private[compiletime] trait ResultsPlatform extends Results { this: DefinitionsPlatform =>
+
+  // TODO: proper error generation
+  protected def reportError(errors: String): Nothing = c.abort(c.enclosingPosition, errors)
```

Comment:
At some point, somewhere, we will have to turn the collection of errors into `String` to display, but we can do it at the very end.